exporter/kafkaexporter/README.md

```diff
@@ -9,11 +9,12 @@ The following settings are required:
 
 The following settings can be optionally configured:
 - `brokers` (default = localhost:9092): The list of kafka brokers
-- `topic` (default = otlp_spans): The name of the kafka topic to export to
-- `encoding` (default = otlp_proto): The encoding of the payload sent to kafka. Available encodings:
-  - `otlp_proto`: the payload is serialized to `ExportTraceServiceRequest`.
-  - `jaeger_proto`: the payload is serialized to a single Jaeger proto `Span`.
-  - `jaeger_json`: the payload is serialized to a single Jaeger JSON Span using `jsonpb`.
+- `topic` (default = otlp): The name of the kafka topic to export to
```

Comment:
This is a breaking change. Make sure to mention in the CHANGELOG.

Reply:
Looking at the current CHANGELOG file, does this breaking have to be associated with a specific version or filed under "Unreleased"